main.py

```diff
@@ -20,7 +20,7 @@
   development phases. This script represents the backbone of this application. 
   This script is called as below:
 
-  usage: main.py [-h] [--version] [--input FILEPATH] [--policy FILENAME] COMMAND
+  usage: main.py [-h] [--version] [--input FILEPATH] [--policy FILENAME] [--command_policy FILENAME] COMMAND
```

Comment:
So, in my mind, and let me know if you guys think differently, but the policy file from last weeks code and the new one Catherine has implemented should be the same. That is to say, the old policy file should just be extended to support the new functionality.